source-academy/frontend
# Recompute goal completion in frontend

### Description

goalProgress.completed is now recomputed when fetching the goal progress from the backend to avoid the goal from being incorrectly completed when goal was previously completed but target count is increased.

### Type of change


### How to test

Create a goal and an achievement that uses that goal. 
Complete the goal.
Increase the goal's target count so that you would not complete the goal.
Return the the achievement dashboard and verify that the achievement and goal are not completed.

### Checklist
